README.md

```diff
@@ -16,7 +16,11 @@ the square brackets (for multiple-choice questions).
 Describe the purpose of a clearfix in CSS, and give an example of how to do it.
 
 Your Answer:
-```text
+```Clearfix is used to make a div expand to enclose all of it's floating child elements
+
+<div class='clearfix'>
+    <div style='float: right;' class='clearfix'>some element</div>
+    </div>
 ```
```

Comment:
only the clearfix class is needed on the containing element. not the floated elements.